universal-login-sdk/test/integration/core/gasPriceOracle.ts

```diff
@@ -0,0 +1,23 @@
+import {expect} from 'chai';
+import {getGasPrices, GasPriceOption} from '../../../lib/integration/http/gasPriceOracle';
+
+describe('INT: GasPriceOracle', () => {
```

Comment:
Tests should execute without a connection to the internet. Mock the http response.

Reply:
1) INT -> UINT